scripts/core/editor3/html/to-html/HTMLGenerator.js

```diff
@@ -160,12 +163,26 @@ export class HTMLGenerator {
      * @description Returns an array of annotations sorted by offset.
      */
     getAnnotations(contentBlock) {
-        let ranges = contentBlock.getData().get('inlineStyleRanges');
+        let annotationIds = [];
+        let ranges = [];
+        let styleRanges = contentBlock.findStyleRanges((charMeta) => {
```

Comment:
Were there any problems using/extending `BlockInlineStyleWrapper` as I've suggested before? Your approach might work, but considering the ease of maintenance, it's better not to have multiple pieces of code doing very similar things.